app/src/main/java/org/jellyfin/androidtv/preferences/ui/UserPreferencesFragment.kt

```diff
@@ -142,6 +143,10 @@ class UserPreferencesFragment : LeanbackSettingsFragmentCompat() {
 				Utils.getVersionString()
 			}
 
+			findPreference<EditTextPreference>("device_model")?.summaryProvider = Preference.SummaryProvider<EditTextPreference> {
+				Build.MODEL
```

Comment:
We might want to add the manufacturer too. Just like the apiclient does:
https://github.com/jellyfin/jellyfin-apiclient-java/blob/master/android/src/main/java/org/jellyfin/apiclient/interaction/AndroidDevice.kt#L24